src/scss/_variables.scss

```
@@ -1,9 +1,15 @@
 /* COLOR GROUP */
-
 $font-color-black: #151515;
 $grey: #969696;
 $color-accent: #ee4c1f;
 $color-accent-darknest: #ca3a12;
 $color-patent-background: #f4f4ff;
 $color-block-background: #fafafd;
+$color-white-bg: linear-gradient(
+  109deg,
+  #ffffff 5%,
+  rgba(255, 255, 255, 0.51) 103%
+);
```

It's gradient, not color
```suggestion
$gradient-white: linear-gradient(
  109deg,
  #ffffff 5%,
  rgba(255, 255, 255, 0.51) 103%
);
```